src/PointwiseFunctions/AnalyticSolutions/GeneralRelativity/KerrHorizon.hpp

```diff
@@ -0,0 +1,28 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+
+namespace gr {
+namespace Solutions {
+
+/*!
+ * \brief Radius of Kerr horizon in Kerr-Schild coordinates.
+ *
+ * \details Computes the radius of a Kerr black hole as a function of
+ * angles.  The input argument `theta_phi` is typically the output of
+ * the `theta_phi_points()` method of a `YlmSpherepack` object; i.e.,
+ * a std::array of two DataVectors containing the values of theta and
+ * phi at each point on a Strahlkorper.
+ */
```

Comment:
Please if possible cite equations in a paper where this comes from. I think I found the code in spec, but I don’t know the reference to cite here.

Reply:
I don't know where this comes from. Presumably one of us (maybe me?) derived it from the Kerr-Schild relations years ago.

Reply:
I just rederived this.  I can put a sketch of the derivation in the documentation.

Reply:
See, for example, equation 15 of https://arxiv.org/pdf/gr-qc/0306056.pdf for a spin aligned with the z-axis.  The equation is simply that of a rotated oblate ellipsoid with the given semi-major axes.

Reply:
pls look at the documentation I put in, which also (hopefully) makes it clear how angular coordinates are defined.